### PR TITLE
Install updated SQLite driver

### LINKED ISSUE #108: Please update sqlite to 3.31.1.
I delete book error. [https://github.com/janeczku/calibre-web/issues/1639](https://github.com/janeczku/calibre-web/issues/1639), please update sqlite to 3.31.1 to resolve it.

### PR DESCRIPTION
**Pull request by Rabattkarte**
closes #108

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [ x ] I have read the [contributing](https://github.com/linuxserver/docker-calibre-web/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

I install `pysqlite3-binary` via `pip` and symlink the shared object (.so) to the app directory as suggested by [upstream dev](https://github.com/janeczku/calibre-web/issues/1639#issuecomment-733885878).

## Benefits of this PR and context:

This fixes #108 by installing `pysqlite3-binary` via `pip`. Deletion of books does work again.

## How Has This Been Tested?

I built containers and tested:
- Adding a book
- Updating metadata
- Deleting a book
- Checking the SQLite version in /stats to be >= 3.31.1

## Source / References:

- https://github.com/janeczku/calibre-web/issues/1639
- #108

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,6 +44,12 @@ RUN \
 	requirements.txt && \
  pip3 install --no-cache-dir -U -r \
 	optional-requirements.txt && \
+ echo "**** install updated SQLite driver ****" && \
+ pip3 install --no-cache-dir -U \
+	pysqlite3-binary && \
+ ln -fs \
+ $(find /usr/local/lib/python3.6/dist-packages/pysqlite3/_sqlite3.*.so) \
```
Comment (aptalca) [inline Dockerfile:51]: When python gets updated to 3.7, this will break

**Comment by aptalca** [comment]
It's failing on arm32

**Comment by Rabattkarte** [comment]
> It's failing on arm32

Yep, I accidentally triggered the build again. I need to look into how to test this locally, I'm a little bit confused by the [CONTRIBUTION instructions here](https://github.com/linuxserver/docker-calibre-web/blob/master/.github/CONTRIBUTING.md#testing-your-changes).

I will look into this later.

**Comment by CHBMB** [comment]
I've managed to build this on a focal base now which should sort out any of the sqlite issues.  I'm just waiting for some upstream stuff to happen before I can open a PR.

**Comment by CHBMB** [comment]
@Rabattkarte Thanks for the PR, appreciate the input, but I've gone a different direction in tackling the issue.  Hope that's ok.  BTW armhf gave me some issues too!

**Pull request closed without merging**